docs/source/getting-started.md

```diff
@@ -5,7 +5,7 @@ Jupyter Kernel.
 
 ### Installation
 
-See [the readme](https://github.com/jupyter-incubator/kernel_gateway/blob/master/README.md)
+See [the readme](https://github.com/jupyter/kernel_gateway/blob/master/README.md)
```

Comment:
Shouldn't this link to the install section of the doc? The README just points there now.